src/factory.js

```diff
@@ -39,6 +40,12 @@ export default function factory(React, empty) {
       })
     }
 
+    const isRedirect = FailureComponent !== undefined
```

Comment:
Haha I was trying to understand why after created a backwards compatible API all the tests were failing. Refactoring is the source of so many bugs 😄